### PR TITLE
8899 task: Add empty circle Icon variant

### PR DESCRIPTION
**Pull request by stuartwilkes**
Relates https://github.com/wellcometrust/corporate/issues/8646
Adds an empty circle Icon, as shown in the designs linked in the above ticket

### CODE REVIEW

File: src/components/Icon/components/IconActionEmptyCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const IconActionCircle = (props: React.SVGAttributes<SVGSVGElement>) => (
```
Comment (creido-welly) [inline src/components/Icon/components/IconActionEmptyCircle.tsx:3]: `props` isn't used anywhere so this can be omitted

**Review by creido-welly — APPROVED** [review]
One minor comment regarding props but otherwise looks good. I've also verified the size matches other icons in the design system using a bit of devtools magic 🚀 

Screenshot 2021-09-02 at 09 53 52

File: src/components/Icon/components/IconActionEmptyCircle.tsx (hunk 1 above)
Comment (stuartwilkes) [reply src/components/Icon/components/IconActionEmptyCircle.tsx:3]: Ah my mistake when copying the amended svg markup - I removed the props. Have added it back in.

**Comment by stuartwilkes** [comment]
> One minor comment regarding props but otherwise looks good. I've also verified the size matches other icons in the design system using a bit of devtools magic 🚀
> 

Thanks - have added the props back in (my mistake from copying the SVG code out of my text editor). Are you happy with the naming? I was in two minds between IconActionEmptyCircle and IconActionCircle - but IconActionEmptyCircle seemed to make most sense?

File: src/components/Icon/components/IconActionEmptyCircle.tsx (hunk 1 above)
Comment (creido-welly) [reply src/components/Icon/components/IconActionEmptyCircle.tsx:3]: No worries. As it happens, I'm toying with removing the props spreading from all icons as everything they really need is governed by the parent Icon component (height, width and fill colour). The spread is convenient but also opens it up to bugs.

**Comment by creido-welly** [comment]
IconActionEmptyCircle works for me. May change it to IconActionCircleEmpty in future to emphasise the variation of the circle  but will advise if that is the case

**Pull request merged**